src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -7177,8 +7182,9 @@ private bool IsPossibleStatement(bool acceptAccessibilityMods)
                 case SyntaxKind.InternalKeyword:
                 case SyntaxKind.ProtectedKeyword:
                 case SyntaxKind.PrivateKeyword:
+                // could be a local function with attributes
+                case SyntaxKind.OpenBracketToken:  // PROTOTYPE reuse this or always allow the OpenBracketToken?
```

Comment:
i would always allow it.

Reply:
Will revisit this when it comes time to remove the prototype comments before merging to master.